libvmaf/src/feature/cambi.c

```diff
@@ -109,6 +116,24 @@ static const VmafOption options[] = {
         .min = 200,
         .max = 4320,
     },
+    {
```

Comment:
Not sure if we need this. The input `width` and `height` should apply to both the reference and distorted videos.

Were you try to mimic what `enc_width` and `enc_height` does? I think these parameters were needed because the w/h used for encoding may be different from `width` and `height` (for example, your input distorted video is upsampled from an encode of a smaller resolution). I don't see a similar situation apply here.

@mafonsoN Do you want to comment on this as well?

Reply:
We had this discussion with Joel and Mariana at some point, here is the conclusion we reached:
"This (optional) parameter is to account for the cases where a video has been scaled before libvmaf is called. If the source was scaled from, say, 480p to 1080p, we need to have an option to supply the 480p."

cc @joelsole, @mafonsoN

Reply:
Ok. So this is symmetric to `enc_width` and `enc_height`? Just to double check.

We can add these.

Reply:
@li-zhi yes, this is added for completeness to handle any use case.

Reply:
@nilfm can we add a test case where `src_width` and `src_height` are exercised?

Reply:
Added in one of the follow-up commits.